### DISCUSSION TITLE
Episode 181: The C++0x Concepts Story with Doug Gregor (Part 2)

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#73] [Episode]
Episode Link: https://adspthepodcast.com/2024/05/10/Episode-181.html

Feel free to write comments or ask questions 🙂

**Comment by kentf** [comment]
In the alternate universe where Bill would have chosen APL, Conor would have been a C++ aficionado! :)

**Reply by codereport** [reply]
Haha, potentially true.

**Comment by pmcgee69** [comment]
Doug Gregor's archetypes at 28:00 seem like interfaces. What am I missing?
